arrow-site/docs/docs/core/README.md

```diff
@@ -234,7 +234,7 @@ Add the dependencies that you want to use:
 
 <dependency>
     <groupId>io.arrow-kt</groupId>
-    <artifactId>arrow-core</artifactId>
+    <artifactId>arrow-core-jvm</artifactId>
```

Comment:
The fix is that we fix the bug in publishing, so this `-jvm` isn't needed anymore.
This should be fixed in the latest alpha release, can you check in the newlly created Maven template?

Reply:
the alpha release isn't published, the CI commit is broken, as the tag has to be updated

Reply:
Just pushed the latest one. I am trying it out

Reply:
It works in the latest Snapshot 1.0.2 only

Reply:
The only problem is that it doesn't load the jvm binaries, so for it to work we still need the `-jvm`. But from i see in sonatype the last snapshot was released in November https://oss.sonatype.org/content/repositories/snapshots/io/arrow-kt/arrow-core/ for `arrow-core`

Reply:
We're not releasing SNAPSHOT anymore, we're instead releasing fixed alpha version to mavenCentral.

Reply:
Strange, afaik for KotlinX Coroutines it's possible to pull the JVM artifact from Maven without `-jvm`.
Users have also reported that this worked for 1.0.0, but broke in 1.0.1

Reply:
Just posting this for everyone, 
the maven support may work with `1.0.2-alpha.43`